core/templates/dev/head/pages/creator_dashboard/view_suggestion_edit_exploration_state_content_modal_directive.html

```diff
@@ -37,7 +37,8 @@ <h3>Review Suggestion</h3>
     <[ errorMessage ]>
   </div>
   <button class="btn btn-default" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-info" ng-if="isEditButtonShown()" ng-click="editSuggestion()">Edit</button>
+  <button class="btn btn-info" ng-if="EditButtonShown" ng-click="editSuggestion()">Edit</button>
+  <button class="btn btn-info" ng-if="isSaveButtonShown" ng-click="saveSuggestion()">Save</button>
```

Comment:
Would both the edit and save button ever be displayed together? If not, please consider making it a single boolean value, which will toggle between the two buttons.

Reply:
ok sir